build.gradle

```diff
@@ -166,11 +166,6 @@ configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driv
                 'org.mongodb.useSocket': System.getProperty('org.mongodb.useSocket', 'false'),
                 'org.mongodb.disableAsync': System.getProperty('org.mongodb.disableAsync', 'false'),
                 'org.mongodb.async.type': System.getProperty('org.mongodb.async.type', 'nio2'),
-
-                'javax.net.ssl.trustStore': System.getProperty('javax.net.ssl.trustStore'),
```

Comment:
These were messing up the connectivity tests: "javax.net.ssl.trustStore" ends up getting set to the empty string and confounds Java's SSLSocketFactory as a result.  I don't see these being relied on anywhere though, since run-tests.sh takes the path just below by setting "ssl.enabled" and project properties.